src/main/java/edu/kis/powp/jobs2d/command/ComplexCommand.java

```diff
@@ -0,0 +1,26 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.DriverCommand;
+
+import java.util.ArrayList;
+
+public class ComplexCommand implements DriverCommand {
+
+    private ArrayList<DriverCommand> commands = null;
```

Comment:
List type (attribute and in the constructor) would be sufficient and would not limit the implementation.